web/html/src/branding/css/base/components/step-progress-bar.scss

```diff
@@ -0,0 +1,105 @@
+.progress-bar-wrapper {
+  display: flex;
+  flex-direction: column;
+  height: calc(100vh - 124px); /* Ensures full height */
```

Comment:
A fixed offset is not a good way to solve this problem because it will break as soon as something else in the layout changes. E.g. if the user has global notifications, I would expect this to break. Can't we use flex layouts on the respective parents to automatically fill out the required space?